manifests/base/controller/deployment.yaml

```diff
@@ -41,10 +41,16 @@ spec:
         ports:
         - containerPort: 3000
           protocol: TCP
+        volumeMounts:
+        - name: patch-file
+          mountPath: ~/
         resources:
           limits:
             cpu: 500m
             memory: 256Mi
           requests:
             cpu: 200m
             memory: 100Mi
+      volumes:
+      - name: patch-file
```

Comment:
Semantics: `patch-file` volume is actually a directory, not a file, change the name please